### DISCUSSION TITLE
Welcome to Tiled Discussions!

### DISCUSSION THREAD

**Discussion by bjorn** [mapeditor/tiled#2948] [General]
## Welcome to the Tiled discussions area on GitHub! 👋

For the past years we've been using the [Tiled Discourse forum](https://discourse.mapeditor.org/) as a place where community members can meet, show off their projects and answer each other's questions. Today, GitHub launched Discussions, and we'd like to give it a try. Of course, this place needs you to have a GitHub account whereas the forum was open to various other social logins. However, it also integrates nicely with the rest of GitHub.

So we’re trying out Discussions! As with the forum, you're welcome to:

* Ask and answer questions.
* Share ideas.
* Talk with other community members.

Please be welcome to others and open-minded!

You can use this topic to introduce yourself, tell us about how you're using Tiled or about what you'd like to find in this community.

**Comment by encelo** [comment]
Hello @bjorn, I'm encelo, the main programmer behind [nCine](https://ncine.github.io/), a cross-platform 2D game framework written in C++11. As I have written in my other [thread](https://github.com/mapeditor/tiled/discussions/2952), I'm currently working on a loader/viewer of Tiled maps so that nCine's users can create nice maps for their games. :wink:
